src/java.base/share/classes/sun/security/provider/SecureRandom.java

```diff
@@ -65,10 +65,14 @@
     private int remCount;
 
     /**
-     * This empty constructor automatically seeds the generator.  We attempt
-     * to provide sufficient seed bytes to completely randomize the internal
-     * state of the generator (20 bytes).  Note, however, that our seed
-     * generation algorithm has not been thoroughly studied or widely deployed.
+     * This empty constructor.
+     * <p>
+     * If the initial state has not been provided by the user via a setSeed()
+     * call, on the first call to engineGetBytes this object will call the
+     * SeedGenerator to provide sufficient seed bytes to completely randomize
+     * the internal state of the generator (20 bytes).  Note that the old
+     * threaded seed generation algorithm is provided only as a fallback, and
+     * has not been thoroughly studied or widely deployed.
      *
      * <p>The first time this constructor is called in a given Virtual Machine,
      * it may take several seconds of CPU time to seed the generator, depending
```

Comment:
In my understanding this is not true. The constructor will finish in no time, it's the first engineBytes() call that will take several seconds. We may say "this auto seeding process will...".

Reply:
Are you talking about the line 78?  Yes, you are correct, I didn't update anything beyond the initial comment.  I can update that as well.

Reply:
Yes, the paragraph starting from line 77.

Reply:
Got it, thanks for the reviews.